features/uploaded-files/member-uploads-files.feature

```diff
@@ -72,3 +72,10 @@ Feature: Member uploads files not associated with a shf application
     And I should not see t("uploaded_files.create.success", file_name: 'biff-image.png')
     When I am on the "my uploaded files" page
     Then I should not see "tred.exe"
+
+  Scenario: Member tries to save without specifying upload file
+    When I am on the "upload a new file" page
+    And I click on t("save")
+    Then I should see t("model_errors.one")
```

Comment:
I think the right thing to do is to disable the save/submit button if no file has been uploaded. So this scenario should be something like:
 When I am on the "upload a new file" page
 Then the link button t("save") should be disabled
 (then after I upload a file, Then the link button t("save") should not be disabled

We should have a scenario where a user tries to upload a file that is empty (size == 0).  (That scenario should go into the `size_validation.feature` file. And the `size_validation.feature` file should be moved into the `features/uploaded-files` directory.)